src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -2359,6 +2383,8 @@ public Response getContentAccessBody(@Verify(Consumer.class) String consumerUuid
                 .build();
         }
 
+        log.info("GENERATING NEW CERTIFICATE!");
```

Comment:
Double checking to see if this log statement and the one on line 2404 were accidentally left in from testing

Reply:
It was, good catch.

Reply:
Fixed